src/core/Akka.Streams/Dsl/Sink.cs

```diff
@@ -270,8 +270,7 @@ public static class Sink
         /// </summary>
         /// <typeparam name="TIn">TBD</typeparam>
         /// <returns>TBD</returns>
-        public static Sink<TIn, Task> Ignore<TIn>()
-            => new Sink<TIn, Task>(new SinkholeSink<TIn>(Shape<TIn>("BlackholeSink"), DefaultAttributes.IgnoreSink));
+        public static Sink<TIn, Task> Ignore<TIn>() => FromGraph(new IgnoreSink<TIn>());
```

Comment:
That would probably take a while :-) and it only helps people that are already familiar with the scala API, but I see the point from a maintainers perspective. 
Maybe we could do it incremental, write down every difference we see while working on/with it?